### PR TITLE
- Added the C++ language to the list.

### PR DESCRIPTION
**Pull request by motasimmakki**

### CODE REVIEW

File: lang/cpp.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+author: motasimmakki
+---
+
+```c++
+#include<iostream>
+int main() {
+    std::cout<< "Hello, World";
+    return 0;
+}
+\```
```
Comment (Panquesito7) [inline lang/cpp.md:14]: Remove the `\` here. 🙂

**Review by Panquesito7 — COMMENTED** [review]
Also, I think this is a duplicate of #41.

File: lang/cpp.md (hunk 1 above)
Comment (motasimmakki) [reply lang/cpp.md:14]: > Also, I think this is a duplicate of #41.

I did not notice that, and #41 is also not merged yet, I think.

Comment (amino19) [reply lang/cpp.md]: @motasimmakki: But the work has already done by @Panquesito7, also he's the moderator of [**DEVSTRONS'**](https://discord.gg/MVujzTBqed) :sunglasses: 

@vsnthdev: ig, we've to consider this spam request, & need to close this!

Comment (motasimmakki) [reply lang/cpp.md]: Okay No problem, happy learning to me!🙃

**Review by vsnthdev — COMMENTED** [review]
This PR is a duplicate of #41 plus both PRs don't contain the full article. Closing this.

**Pull request closed without merging**